### DISCUSSION TITLE
Phalcon future and sponsoring

### DISCUSSION THREAD

**Discussion by markofo** [phalcon/cphalcon#16356] [General]
Dear Phalcon devs,

first of all, thank you for developing the Phalcon framework which we love.

We are building a product on it and we would like to sponsor further Phalcon development. However it seems that there has been not so much of a progress lately. What is the future of Phalcon development? Are you working on Phalcon 6, and is there a future at all?

Thank you for answering!
Marek

**Comment by niden** [comment]
No/little activity = we are busy in our personal lives/holidays etc.

There is work on v6, not as much as I would like but trying to get there. In the meantime a handful of bugs have been fixed for v5. I am hoping for a release soon.

There is always future so long as people use the framework and can can code it :)

**Reply by markofo** [reply]
As long as there will be zephir, the community that can code phalcon will be very limited I am afraid. I am hoping that you will manage to get to v6 soon and then also our team will be able to contribute. 

For the time being, we have to do workarounds for issues with ORM that were not present in phalcon v3 and are coming from zephir.

We have became a sponsor and we are looking forward to work with phalcon further :)
